src/ResourceManager/Sql/Commands.Sql/Commands.Sql.csproj

```diff
@@ -447,10 +452,21 @@
       <Project>{04B53727-16E6-4EF0-9116-04C7F1C33842}</Project>
       <Name>SqlManagement</Name>
     </ProjectReference>
+<<<<<<< HEAD
```

Comment:
resolve merge conflict

Reply:
Weird.... I resolved this but it didn't save. Resolved now.